### PR TITLE
Update snapshot-pvc.md

### PR DESCRIPTION
**Pull request by cristianrat**
I've been got by this very issue
Not once, but twice - I was lazy, I though it would take months for my disk to fill, like the first time, but the volume of data increased so much that it didn't
This is the fix I used in my case

### CODE REVIEW

File: docs/managing-dragonfly/operator/snapshot-pvc.md [hunk 1]
```diff
@@ -80,3 +80,23 @@ pod "redis-cli" deleted
 ```
 
 You should see the value `bar` for the key `foo`. This means that the data was restored from the snapshot that was stored in the PVC.
+
+### NOTE:
```
Comment (Niennienzz) [inline docs/managing-dragonfly/operator/snapshot-pvc.md:84]:
```suggestion
## Managing Disk Space for Continuous Snapshots

```

File: docs/managing-dragonfly/operator/snapshot-pvc.md [hunk 2]
```diff
@@ -80,3 +80,23 @@ pod "redis-cli" deleted
 ```
 
 You should see the value `bar` for the key `foo`. This means that the data was restored from the snapshot that was stored in the PVC.
+
+### NOTE: 
+snapshots are not pruned and may fill up your disk. If you do not specify a snapshot name, pruning is something you need to manage yourself.
```
Comment (Niennienzz) [inline docs/managing-dragonfly/operator/snapshot-pvc.md:85]:
```suggestion
Snapshots are not automatically pruned by Dragonfly, and they may fill up your disk. If you do not specify a snapshot name, pruning is something you need to manage yourself.
```

File: docs/managing-dragonfly/operator/snapshot-pvc.md [hunk 3]
```diff
@@ -80,3 +80,23 @@ pod "redis-cli" deleted
 ```
 
 You should see the value `bar` for the key `foo`. This means that the data was restored from the snapshot that was stored in the PVC.
+
+### NOTE: 
+snapshots are not pruned and may fill up your disk. If you do not specify a snapshot name, pruning is something you need to manage yourself. 
+Specifying a snapshot name would avoid this problem by only ever having one snapshot.
```
Comment (Niennienzz) [inline docs/managing-dragonfly/operator/snapshot-pvc.md:86]:
```suggestion
Specifying a snapshot name would avoid this problem by only ever having one snapshot.

```

File: docs/managing-dragonfly/operator/snapshot-pvc.md [hunk 4]
```diff
@@ -80,3 +80,23 @@ pod "redis-cli" deleted
...
+```
```
Comment (Niennienzz) [inline docs/managing-dragonfly/operator/snapshot-pvc.md:87]:
```suggestion
```

File: docs/managing-dragonfly/operator/snapshot-pvc.md [hunk 5]
```diff
@@ -80,3 +80,23 @@ pod "redis-cli" deleted
...
+  name: dragonfly-pvc
+spec:
+  replicas: 1
+  args: ["--dbfilename=mySnapshotName"]
+  snapshot:
+    ...
+    ...
+EOF
+```
+```
```
Comment (Niennienzz) [inline docs/managing-dragonfly/operator/snapshot-pvc.md:102]:
```suggestion

```

**Review by Niennienzz — COMMENTED** [review]
The content looks good to me, and thanks to the contribution!

Added some styling adjustments.

**Review by Niennienzz — APPROVED** [review]
LGTM.

**Comment by cristianrat** [comment]
afraid I don't have the power :)
```This branch has no conflicts with the base branch
Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests.
```

**Pull request merged**